testomania/src/main/java/com/earth/testomania/home_screen/presentation/HomeScreen.kt

```diff
@@ -195,46 +191,52 @@ fun HomeScreenContent(
 
 @Composable
 fun CardButton(
-    destinationInfo: HomeDestinations,
+    destinationInfo: HomeDestinationItem,
     navigator: DestinationsNavigator? = null,
     scaffoldState: ScaffoldState,
     modalBottomSheetState: ModalBottomSheetState,
     networkConnectivityObserver: NetworkConnectivityObserver,
-    ) {
+) {
     val scope = rememberCoroutineScope()
     val comingSoonStr = stringResource(id = R.string.coming_soon)
     val dismissStr = stringResource(id = R.string.dismiss)
     val viewModel: HomeScreenViewModel = hiltViewModel()
     val status by networkConnectivityObserver.observe()
         .collectAsState(initial = false)
 
-    SurfaceCard(modifier = Modifier.size(125.dp), shape = RoundedCornerShape(10.dp), enabled = status == ConnectivityObserver.ConnectionState.Available, onClick = {
-        dismissCurrentSnackbar(scaffoldState)
+    SurfaceCard(
+        modifier = Modifier.size(125.dp),
+        shape = RoundedCornerShape(10.dp),
+        enabled = status == ConnectivityObserver.ConnectionState.Available,
+        onClick = {
+            dismissCurrentSnackbar(scaffoldState)
 
-        when (destinationInfo.destination?.route ?: destinationInfo.destinationWithParam?.route) {
-            SKILLZ_ROUTE, DUMMY_ROUTE -> {
-                scope.launch {
-                    scaffoldState.snackbarHostState.showSnackbar(
-                        message = comingSoonStr,
-                        actionLabel = dismissStr,
-                    )
-                }
-                return@SurfaceCard
-            }
-            ABOUT_ROUT -> {
-                viewModel.onBottomSheetPageChange(BottomSheetScreen.About)
-                scope.launch {
-                    modalBottomSheetState.show()
-                }
-            }
-            "$TECHNICAL_ROUTE/ALL" -> {
-                viewModel.onBottomSheetPageChange(BottomSheetScreen.Technical)
-                scope.launch {
-                    modalBottomSheetState.show()
-                }
-            }
-        }
-    }) {
+            navigator?.navigate(destinationInfo.destination ?: return@SurfaceCard)
+
+//        when (destinationInfo.destination?.route ?: destinationInfo.destinationWithParam?.route) {
+//            SKILLZ_ROUTE, DUMMY_ROUTE -> {
+//                scope.launch {
+//                    scaffoldState.snackbarHostState.showSnackbar(
+//                        message = comingSoonStr,
```

Comment:
also remove this or add TODO

Reply:
Yes i think we do not need now categories that are not implemented, I think we can remove this